python/lsst/ctrl/bps/parsl/sites/torque.py

```diff
@@ -75,6 +75,11 @@ class Torque(SiteConfig):
       script (each line usually starting with ``#PBS``).
     """
 
+    def __init__(self, *args, **kwargs):
+        # Have BPS-defined resource requests for each job passed to executor.
+        kwargs["resource_list"] = ["priority"]
```

Comment:
Could it be made to have a default resource_list if none given?  Or is it just a coincidence that all of the sites needed to have this added?

Reply:
I thought about making that the default, but it seemed better to be explicit since that default assumes that a particular parsl Executor class is used, i.e., the HighThroughputExecutor.   I would also like to add a recently added executor in parsl: [TaskVineExecutor](https://github.com/Parsl/parsl/blob/73bb4f2f86cdedf0976ed70d6ab4f297195f2ba1/parsl/executors/taskvine/executor.py#L54) which is supposed to be the successor to WorkQueueExecutor, so that will have a different set of resources to access.